com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs

```diff
@@ -266,17 +295,13 @@ public static bool IsGameCamera(Camera camera)
         /// </summary>
         /// <param name="camera">Camera to check state from.</param>
         /// <returns>Returns true if the given camera is rendering in stereo mode, false otherwise.</returns>
+        [Obsolete]
```

Comment:
Fix in this PR: Add a message to Obsolete attribute explaining what to use instead.